_components/page-templates/page-templates.html

```diff
@@ -0,0 +1,102 @@
+---
+layout: styleguide
+title: Page templates
+category: Page templates
+permalink: /page-templates/
+lead: These templates for a landing page and a documentation page are built from some of the components of the Draft U.S. Web Design Standards. Each of these pages is flexible; you can add or remove components or sub-components to suit your needs. These templates are great starting point for prototyping or for trying the Standards to see if they’re a good fit for you.
+maturity: alpha
+---
+
+<h2 class="usa-heading" id="landing-page">Template 1: Landing page</h2>
+<p>A landing page is intended to provide someone’s first impression of your agency or program. Often, they’ll arrive from somewhere else without much context, like a search result or a colleague’s email. So a landing page needs to be clear, engaging, and contextualizing.</p>
+<div class="preview">
+  <a class="media_link" href="{{ site.baseurl }}/page-templates/landing/">
+    <img src="{{ site.baseurl }}/assets/img/templates/template-landing.png" alt="Basic header">
+  </a>
+  <a class="usa-button" href="{{ site.baseurl }}/page-templates/landing/">Demo landing page template</a>
+</div>
+{% include code/accordion.html component="header-basic" %}
```

Comment:
Ah! Dis-regard. This is the actual page template and an include for the header 👍